### PR TITLE
Makes Mosins bulky, lets you saw them in Obrezes

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
Makes the Mosin Nagant rifle bulky instead of normal sized, but allows you to saw them off much like your average shotgun to normal size. However, sawing it down will introduce bullet spread because of the drastically reduced barrel.

Sprites:

Accuracy when sawed off (distance of 6 tiles, 4 hits out of 8 shots):

## Why It's Good For The Game
The Mosin Nagant is a really fucking big gun, 48.5 inches in its full length model, and with a bayonet attached it can reach near 6 feet long in total. Item size isn't 1:1 with real life, but considering just how notorious they are for being really big, paying homage to it feels right. Plus, running around with sawed off guns feel much more illegal and thus fun.

## Changelog
:cl: Shaps/Ryll
tweak: Mosin Nagants are now bulky, but can be sawed off to fit in a bag
/:cl:

### CODE REVIEW

**Comment by Eaglendia** [comment]
The change this weapon needed. :+1:

**Comment by necromanceranne** [comment]
Having shitty sawn-off bolt action pistols is goddamn funny and I love it.

**Comment by HomocidlePancakes** [comment]
B-but... My NukeOp memes...

**Comment by zxaber** [comment]
Relevant video
https://www.youtube.com/watch?v=C7VSu_zdXEg

**Comment by 81Denton** [comment]
Since this makes the weapon far less viable to use, feel free to drop the nuke op uplink TC cost from two to one.

Path is `/datum/uplink_item/dangerous/bolt_action`

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -59,6 +61,30 @@ obj/item/gun/ballistic/rifle/attackby(obj/item/A, mob/user, params)
 	knife_x_offset = 27
 	knife_y_offset = 13
 
+/obj/item/gun/ballistic/rifle/boltaction/attackby(obj/item/A, mob/user, params)
+	..()
+	if(istype(A, /obj/item/circular_saw) || istype(A, /obj/item/gun/energy/plasmacutter))
```
Comment (CPTANT) [inline code/modules/projectiles/guns/ballistic/rifle.dm]: I severely dislike checking for specific items instead of an item attribute.

**Comment by wesoda25** [comment]
idk why but as late bagils cargo meta has been to order a ton of mosin’s, and they are honestly really good at shutting down any threat if a pack of people with mosins are going after it. Think this might help deal with that

**Comment by Shapsy** [comment]
i fucked it and did this from master

**Pull request closed without merging**